validation/README.md

```diff
@@ -0,0 +1,29 @@
+# Schema Validation
+
+In addition to the [import tool](https://github.com/datacommonsorg/import), we do further validations of schema.
+
+## Install requirements
+
+```
+pip install -r requirements.txt
+```
+
+## URL Checker
+
+We check that all new urls referenced in schema are valid (i.e. return a 200 status code). To run the url checker: 
+
+```
+export DC_API_KEY=<AUTOPUSH KEY>
+python3 url_checker.py
+```
+
+Please fix all invalid urls. (Note that redirects are considered invalid and should be replaced by the final url or otherwise updated). By default the checker will only process modified files.
```

Comment:
Is it feasible to consider the redirect thing not invalid?  Because I feel there might be legitimate redirects of a shorter/top-level URL redirecting to something else.

Reply:
I agree that some redirects might be useful, but the original bug was to catch redirects, so for now this will at least alert that they exist (and if we decide to keep some, we can just put it in the allowlist). Or if we'd like to finetune which redirects to accept vs flag, I can do that as a follow up

Reply:
sg